pkg/Microsoft.NETCore.Platforms/runtimeGroups.props

```diff
@@ -111,6 +118,12 @@
       <Versions>16.04;16.10</Versions>
       <TreatVersionsAsCompatible>false</TreatVersionsAsCompatible>
     </RuntimeGroup>
+    <RuntimeGroup Include="ubuntu">
```

Comment:
Why aren't these versions just added to the 16.x versions of Ubuntu?

Reply:
Because of how the file was originally structured. It's using `TreatVersionsAsCompatible` and has the `ubuntu releases are compatible on major versions` comment. So I assume it's split to maintain that.

Reply:
That's not right.  You can create another item without setting `<TreatVersionsAsCompatible>false</TreatVersionsAsCompatible>`.  What you've done here is add 17.04 and 17.10, but 17.10 won't be importing 17.04.  This is different than what was done between 15 and 16 and I don't see a good reason for it or an explanation of why it would be different.

Reply:
Actually TreatVersionsAsCompatible=false makes all versions independent.  The reason for the split between 14+15 and 16 was that 16 added a new architecture: `arm64`.   Since you aren't adding a new arch you can just include this with the 16.x versions.  You can try it and examine the diff to prove that it doesn't make a difference in the generated files.

Reply:
@ericstj I'll update the PR with that change then.

Reply:
Also I think that comment ubuntu comment about major versions is incorrect 🙁 .  If you'd like you can delete it or correct it to say all ubuntu versions are independent.